AmplifyTools/amplify-tools.sh

```diff
@@ -0,0 +1,192 @@
+#!/bin/sh
+
+# Copyright Amazon.com Inc. or its affiliates.
+# All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+###########################################################################################
+#                                                                                         #
+# WARNING: AmplifyTools has been deprecated.                                              #
+#                                                                                         #
+# More info: https://github.com/aws-amplify/amplify-ios/blob/main/AmplifyTools/README.md  #
+#                                                                                         #
+###########################################################################################
+
+echo "warning: AmplifyTools has been deprecated. More info: https://github.com/aws-amplify/amplify-ios/blob/main/AmplifyTools/README.md"
```

Comment:
Note: deprecation warning will show up in Xcode for customers using AmplifyTools

Reply:
Another thought: When we're actually ready to retire it, let's change this script to output an `error:` message with the deprecation note, and do nothing else. That way customers who didn't remove the custom build phase will see an actionable error item in their build output, instead of a "missing script" error. Some time after that, once new version uptake is sufficiently high, we can remove the script altogether.